app/views/includes/scripts.html

```diff
@@ -3,6 +3,7 @@
   <script src="{{scriptUrl}}"></script>
 {% endfor %}
 
+<script src="/public/javascripts/kit.js"></script>
```

Comment:
When I was working on the "smallest possible kit" I found it surprisingly helpful to include a `govuk-prototype-kit.config.json` in the kit itself - that way you can use the extensions mechanism to include scripts etc.

https://github.com/alphagov/govuk-prototype-kit/blob/smallest-possible-prototype-kit-boilerplate_companion/govuk-prototype-kit.config.json

Reply:
Not actually possible when we're duel running as a module and not as a module.